src/main/java/redis/embedded/core/ExecutableProvider.java

```diff
@@ -25,8 +30,18 @@ default File get() throws IOException {
         return executable.isFile() ? executable : writeResourceToExecutableFile(executablePath);
     }
 
+    /*
+    TODO: Decide whether the custom path should have priority or not.
+    At the moment it breaks redis.embedded.RedisServerTest.shouldOverrideDefaultExecutable
+     */
     static ExecutableProvider newRedis2_8_19Provider() {
-        return newRedis2_8_19Map()::get;
+        return osArchitecture -> {
+            if (CUSTOM_PATH != null) {
```

Comment:
IMO if a custom path is specified then it should be preffered.
But there is a unit test that breaks due to this new behavior.
Up to you to decide whether to drop the test or to use the CUSTOM_PATH as a fallback.

Reply:
I made the whole thing configurable. So now it is up to the user to decide what kind of ExecutableProvider they want.